app/HomeScreen.tsx

```diff
@@ -42,31 +49,50 @@ function HomeScreen() {
   const [selected, setSelected] = useState<string>('');
   const [schoolOptions, setSchoolOptions] = useState<SchoolKeyPair[]>([]);
   const [dropdownStyle, setDropdownStyle] = useState<object>(styles.dropdownUnselected);
+  const [resourceURLs, setResourceURLs] = useState<ResourceURLs | null>(null);
+  const [missions, setMissions] = useState<Entry[] | null>(null);
+
+  // Fetch resource URLs from Firebase
+  useEffect(() => {
+    const fetchResourceURLs = async () => {
+      const urls = await getResourceURLs();
+      if (urls) {
+        setResourceURLs(urls);
+      }
+    };
+    fetchResourceURLs();
+  }, []);
+
+  // Fetch mission entries from Firebase
+  useEffect(() => {
```

Comment:
I don't think these and the state are needed since you call this in MissionScreen.tsx